ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -252,8 +241,12 @@
 
   private String streamName;
 
+  private List<ServiceContext> serviceContexts = new ArrayList<>();
```

Comment:
It looks like all usages are `contains`, why not use a `Set`?

Reply:
Right. It's better to use `Set`